data/colors/dark.ts

```diff
@@ -601,7 +601,7 @@ export default {
   underlinenav: {
     border: alpha(get('scale.gray.6'), 0),
     borderHover: get('scale.gray.6'),
-    borderActive: '#f78166',
+    borderActive: get('scale.coral.3'),
```

Comment:
I think this needs to be changed back to the hex value since the coral scale was only added to `colors_v2` and doesn't exist in v1.

Reply:
👍